jsonnet/components/thanos-querier.libsonnet

```
@@ -408,27 +389,25 @@ function(params)
                 ],
               },
               {
-                name: 'oauth-proxy',
-                image: 'quay.io/openshift/oauth-proxy:latest',  //FIXME(paulfantom)
+                // This proxy takes over the old oauth proxy and is used to proxy prometheus metrics as well as queries from Admin console.
```

I wouldn't mention the previous existence of OAuth proxy.
 
```suggestion
                // This proxy is used to proxy requests from the Admin console to the Thanos API.
```